src/icons/SealWallet.tsx

```diff
@@ -7,11 +9,14 @@ import classnames, {
   strokeWidth,
 } from 'classnames/tailwind'
 
-const svgClasses = classnames(
-  boxShadow('shadow-lg'),
-  boxShadowColor('shadow-secondary'),
-  borderRadius('rounded-full')
-)
+const showOnMd = displayFrom('sm')
```

Comment:
I think you can get rid of this proxy `const`

Reply:
Did